storage-node/src/commands/dev/sync.ts

```diff
@@ -56,13 +61,14 @@ export default class DevSync extends Command {
 
   async run(): Promise<void> {
     const { flags } = this.parse(DevSync)
-
+    const bucketId = '' + flags.bucketId
```

Comment:
```suggestion
    const bucketId = flags.bucketId.toString()
```

I think better to use `.toString()` method?